rotkehlchen/db/evmtx.py

```diff
@@ -36,7 +36,8 @@
 
 TRANSACTIONS_MISSING_DECODING_QUERY = (
     'evmtx_receipts AS A LEFT OUTER JOIN evm_tx_mappings AS B ON A.tx_hash=B.tx_hash '
-    'AND A.chain_id=B.chain_ID WHERE B.tx_hash is NULL'
+    'AND A.chain_id=B.chain_ID LEFT JOIN evm_transactions AS C on '
```

Comment:
Why was B.tx_hash is NULL removed? In general why the change here? 

Reading later: Is it due to the filter you have created?

Reply:
Yeah, because it is checked in the filter
![image](https://user-images.githubusercontent.com/63492346/205656914-0c659671-7ca9-45a5-94dc-dd2012055658.png)

Reply:
tx_hash is not unique